docs/resources/users_permission.md

```diff
@@ -0,0 +1,89 @@
+---
+page_title: "keycloak_users_permissions Resource"
+---
+
+# keycloak_users_permissions
+
+Allows you to manage all users Scope Based Permissions.
+
+This is part of a preview keycloak feature. You need to enable this feature to be able to use this resource.
+More information about enabling the preview feature can be found here: https://www.keycloak.org/docs/latest/server_installation/#profiles
+
+When enabling Users Permissions, Keycloak does several things automatically:
+1. Enable Authorization on build-in realm-management client
+1. Create scopes "view", "manage", "map-roles", "manage-group-membership", "impersonate", "user-impersonated"
+1. Create a resource representing the users permissions
+1. Create all scope based permission for the scopes and openid client resource
+
+If the realm-management Authorization is not enable, you have to ceate a dependency (`depends_on`) with the policy and the openid client.
```

Comment:
Is this sentence also correct?  I noticed it was borrowed from the other docs as well.

Sorry, I am not very familiar with this feature of Keycloak, so you'll have to help me out here 😅

Reply:
replaced by "1. Create all scope based permission for the scopes and users resource"